src/MediaWiki/Hooks/SkinTemplateNavigation.php

```diff
@@ -45,6 +45,7 @@ public function __construct( SkinTemplate &$skinTemplate, array &$links ) {
 	public function process() {
 
 		if ( $this->skinTemplate->getUser()->isAllowed( 'purge' ) ) {
+			$skin->getOutput()->addModules( 'ext.smw.purge' );
```

Comment:
I suspect that this may be very wrong. I guess this is as far as imitation can get me. :(